boards/BMS/Src/App/states/App_ChargeState.c

```diff
@@ -44,23 +52,31 @@ static void ChargeStateRunOnTick100Hz(struct StateMachine *const state_machine)
             has_charger_faulted = App_Charger_HasFaulted(charger);
         }
 
-        uint8_t    segment = 0U;
-        uint8_t    cell    = 0U;
-        const bool has_reached_max_v =
-            App_Accumulator_GetMaxVoltage(accumulator, &segment, &cell) > MAX_CELL_VOLTAGE_THRESHOLD;
+        if (is_charger_disconnected || has_charger_faulted)
```

Comment:
Are we broadcasting these messages over CAN? If not we should be

Reply:
yup